internal/pkg/util/fs/mount/mount_linux.go

```diff
@@ -306,13 +306,15 @@ func GetSizeLimit(options []string) (uint64, error) {
 }
 
 // GetKey returns key value for image options
-func GetKey(options []string) (string, error) {
+func GetKey(options []string) ([]byte, error) {
 	for _, opt := range options {
 		if strings.HasPrefix(opt, "key=") {
-			return strings.TrimPrefix(opt, "key="), nil
+			// TODO(mem): base64 encode the key?
```

Comment:
You might need to base64 encode here. Otherwise, "," or "." in the key may create problems.

Reply:
Thanks, fixed!